### PR TITLE
[@types/koa-cors] Add type definitions

### PR DESCRIPTION
**Pull request by romain-faust**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/koa-cors/koa-cors-tests.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import * as Koa from 'koa';
+import * as KoaCors from 'koa-cors';
```
Comment (plantain-00) [inline types/koa-cors/koa-cors-tests.ts]: `KoaCors` is a function here, so it should be `import = require`, same as `Koa`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/koa-cors/koa-cors-tests.ts (hunk 1 above)
Comment (romain-faust) [reply types/koa-cors/koa-cors-tests.ts]: You're right, I make the change ASAP

Comment (romain-faust) [reply types/koa-cors/koa-cors-tests.ts]: Done

**Review by plantain-00 — APPROVED** [review]

**Comment by danwbyrne** [comment]
@romain-faust Hi! I was just scanning this PR as I am making a similar one for `koa-better-body`, and I was just curious if [this line](https://github.com/romain-faust/DefinitelyTyped/blob/63467a86e6ccb05c2dfcf1e5dcb1072b48127b8d/types/koa-cors/index.d.ts#L13) was intentional, or if it should be:
```
credentials?: boolean;
```

**Comment by romain-faust** [comment]
Hi @danwbyrne, yes it is, [see the doc](https://github.com/evert0n/koa-cors#credentials) :)

**Pull request merged**